source/WrappedText.cpp

```diff
@@ -148,8 +163,26 @@ int WrappedText::Height() const
 // Draw the text.
 void WrappedText::Draw(const Point &topLeft, const Color &color) const
 {
-	for(const Word &w : words)
-		font->Draw(text.c_str() + w.Index(), w.Pos() + topLeft, color);
+	if(words.empty())
+		return;
+	
+	if(truncate == Font::TRUNC_NONE)
+		for(const Word &w : words)
+			font->Draw(text.c_str() + w.Index(), w.Pos() + topLeft, color);
+	else
+	{
+		// Apply the truncation to a word only if a line has a single word.
```

Comment:
A single word? or more than one?

Reply:
A single word.
If there are multiple word in a line, it does not try to truncate the line.
Too complicated.